src/extensions/sonarcloud/vss-extension.json

```diff
@@ -2,7 +2,7 @@
   "manifestVersion": 1,
   "id": "sonarcloud",
   "name": "SonarCloud",
-  "version": "1.46.0",
+  "version": "5.50.11",
```

Comment:
afaik, we can bump the task version without needing to bump extension version and it should still bust the task cache properly.

I wanted to suggest that we could implement bumping the patch version per run. and when we release we bump minor and reset the patch back to 0

Reply:
my guess is that the assumption was by deleting and re-installing the extension for ITs , it was clearing the cache.

Reply:
you would think the extension cache would supersede the task cache but it seems that's not the case